#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_filter.dpatch by Michael Meskes <meskes@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add Debian related stuff to filter.pot

@DPATCH@

--- a/gen/filter.pot
+++ b/gen/filter.pot
@@ -13,6 +13,10 @@
 msgid "swaps visible and invisible text"
 msgstr ""
 
+#: modules/filter/debctrl-filter.info:7
+msgid "filter for Debian packaging control files"
+msgstr ""
+
 #: modules/filter/email-filter.info:7
 msgid "filter for skipping quoted text in email messages"
 msgstr ""
@@ -89,6 +93,10 @@
 msgid "mode to check any lines starting with a #"
 msgstr ""
 
+#: modules/filter/modes/debctrl.amf:5
+msgid "mode for Debian packaging control files"
+msgstr ""
+
 #: modules/filter/modes/email.amf:5
 msgid "mode for skipping quoted text in email messages"
 msgstr ""
